content/docs/04.5-devx/03-cluster-groups.md

```diff
@@ -101,6 +101,61 @@ To delete a cluster group, select `Delete Cluster form the **Settings** and ente
 * Azure
 * AKS
 * VMWare
+* OpenShift
+
+<Tabs>
+<Tabs.TabPane tab="Configure OpenShift" key="Configure OpenShift">
+
+#### Configure OpenShift 
+
+<br />
+
+To deploy a virtual cluster on OpenShift:
+
+
+1. Create a new Cluster Group or edit an existing one and click **Settings**.
+2. Select **Settings** in the **Cluster Group Settings** pane.
+3. In the **Advanced Config** file, locate the section ``#Configure SecurityContext of the containers in the VCluster pod``.
```

Comment:
We should not use a comment as an anchor point as these can change between releases. We are better off pointing to the actual parameter.

Reply:
@karl-cardenas-coding The doc now refers to the SecurityContext section.